controls/RHEL-07-040333.rb

```diff
@@ -31,8 +31,8 @@
 If the value is returned as “no”, the returned line is commented out, or no output is returned, this is a finding.'
```

Comment:
This does not match the title of the control
`title 'RHEL-07-040333 - The SSH daemon must not allow authentication using RSA rhosts authentication.'`
This setting should be 'no'